pkg/snapstore/abs_snapstore.go

```diff
@@ -51,14 +52,15 @@ type ABSSnapStore struct {
 }
 
 type absCredentials struct {
-	BucketName     string `json:"bucketName"`
-	SecretKey      string `json:"storageKey"`
-	StorageAccount string `json:"storageAccount"`
+	BucketName     string  `json:"bucketName"`
```

Comment:
A similar unused field is present here: as well https://github.com/gardener/etcd-backup-restore/blob/b311db7f4c9692b4f887319a4fa115fcba5c7e02/pkg/snapstore/s3_snapstore.go#L53-L62

We can get rid of such unused fields in another PR, since I feel touching this part of the code is out of this PR's scope.

Reply:
right. we can do it later then. 👍